src/main/kotlin/com/muxiu1997/mxrandom/metatileentity/MTELargeMolecularAssembler.kt

```diff
@@ -133,24 +140,25 @@ class LargeMolecularAssembler :
         extraTier--
       }
       var times = 2
-      // Subsequent Overclocks Double the number of Jobs finished at once to a Max of 256"
-      while (times < 256) {
+      // Subsequent Overclocks Double the number of Jobs finished at once to a Max of Integer.MAX
+      while (times > 0) {
```

Comment:
Why not write this as (pseudocode)
```kt
var limitedTimes = min(times, extraTier);
times = times * pow(2, limitedTimes);
craftingEUt = craftingEUt * pow(4, limitedTimes);
```
to avoid the loop